aten/src/ATen/native/cuda/AbsKernel.cu

```diff
@@ -8,29 +8,34 @@ namespace at { namespace native {
 
 // We manually overload abs because std::abs does not work with thrust::complex types and ROCm.
 template<typename scalar_t>
-__host__ __device__ static inline scalar_t abs_wrapper(scalar_t v) {
+__device__ static inline scalar_t abs_wrapper(scalar_t v) {
   return ::abs(v);
 }
 
 template<typename T>
-__host__ __device__ static inline c10::complex<T> abs_wrapper(c10::complex<T> v) {
+__device__ static inline c10::complex<T> abs_wrapper(c10::complex<T> v) {
   return std::abs(v);
 }
```

Comment:
These two functions can now be unified as:
```C++
template<typename scalar_t>
__device__ static inline scalar_t abs_wrapper(scalar_t v) {
  return std::abs(v);
}
```
`std::abs` is now supported on ROCm. See: https://github.com/pytorch/pytorch/pull/44804